examples/plot_tensorflow.py

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""
+=================================
+An example for tensorflow
+=================================
+"""
+# %%
+
+import pandas as pd
+from sklearn.datasets import fetch_openml
+from sklearn.metrics import accuracy_score, precision_score, recall_score
+import scikeras
+import warnings
+from tensorflow import get_logger
+import numpy as np
+from scikeras.wrappers import KerasClassifier
+from tensorflow import keras
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+import sklearn.metrics as skm
+%matplotlib inline
+import fairlearn
+from fairlearn.metrics import selection_rate
+
+from fairlearn.metrics import (
+    MetricFrame,
+    false_positive_rate,
+    true_positive_rate,
+    selection_rate
+)
+
+get_logger().setLevel('ERROR')
+warnings.filterwarnings("ignore", message="Setting the random state for TF")
+
+
+data = fetch_openml(data_id=1590, as_frame=True)
+X_raw = data.data
+Y = (data.target == '>50K') * 1
+sex = data.data["sex"]
+
+X = X_raw.drop(labels=['sex'], axis=1)
+X = pd.get_dummies(X)
+
+def get_clf(meta, hidden_layer_sizes, dropout):
+    n_features_in_ = meta["n_features_in_"]
+    n_classes_ = meta["n_classes_"]
+    model = keras.models.Sequential()
+    model.add(keras.layers.Input(shape=(n_features_in_,)))
+    for hidden_layer_size in hidden_layer_sizes:
+        model.add(keras.layers.Dense(hidden_layer_size, activation="relu"))
+        model.add(keras.layers.Dropout(dropout))
+    model.add(keras.layers.Dense(1, activation="sigmoid"))
+    return model
+
+
+clf = KerasClassifier(
+    model=get_clf,
+    loss="binary_crossentropy",
+    hidden_layer_sizes=(100,),
+    dropout=0.5,
+)
+
+clf.fit(X, Y);
+
+y_pred = clf.predict(X)
+
+metrics = {
+    'accuracy': accuracy_score,
+    'precision': precision_score,
+    'recall': recall_score,
+    'false positive rate': false_positive_rate,
+    'true positive rate': true_positive_rate,
+    'selection rate': selection_rate,
+    'count': lambda Y, y_pred: len(Y),
+    'average precision':skm.average_precision_score,
+    'roc-auc': skm.roc_auc_score,
+}
+
+metric_frame = MetricFrame(metrics,

```

Comment:
Given that `MetricFrame` only cares about a column of inputs, I'm not really sure that this is really showing TensorFlow being used with `fairlearn`. For that, I think you'd need to show it working with the mitigation algorithms

Reply:
Yeah, alright then, I will soon try working it out on the mitigation algorithms.

Reply:
I think the question of example notebooks like that is a bit bigger, since there is the question of having TensorFlow installed (which we don't usually). I would suggest removing this file from the PR for now, and making this PR about updating the `test_tensorflow.py` file in the `test_othermlpackages` directory to use the newer wrappers.

Reply:
Alright! I will do as suggested and keep this PR only to update the `test_tensorflow.py` file in the `test_othermlpackages` directory.